src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/abbreviationaswordinname/Example7.java

```diff
@@ -0,0 +1,23 @@
+/*
+AbbreviationAsWordInName
+allowedAbbreviationLength = (default)3
+allowedAbbreviations = ALLOWED
+ignoreFinal = (default)true
+ignoreStatic = (default)true
+ignoreStaticFinal = false
+ignoreOverriddenMethods = (default)true
+tokens = (default)CLASS_DEF, INTERFACE_DEF, ENUM_DEF, ANNOTATION_DEF, ANNOTATION_FIELD_DEF, \
+         PARAMETER_DEF, VARIABLE_DEF, METHOD_DEF, PATTERN_VARIABLE_DEF, RECORD_DEF, \
+         RECORD_COMPONENT_DEF
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.naming.abbreviationaswordinname;
+
+class Example7 {
+  int counterXYZ = 1;
+  final int customerID = 2;
+  static int nextID = 3;
```

Comment:
Violations are expected, there is mistake in config, length= 0

Reply:
Are you sure? For me it looks okay.
```xml
<module name="AbbreviationAsWordInName">
    <property name="allowedAbbreviations" value="ALLOWED"/>
    <property name="ignoreStaticFinal" value="false"/>
</module>
```
```java
public class MyClass {
    public int counterXYZ = 1;                // OK
    public final int customerID = 2;          // OK
    public static int nextID = 3;             // OK
    public static final int MAX_ALLOWED = 4;  // OK, abbreviation is allowed
}
```
![Screenshot from 2023-06-14 09-21-09](https://github.com/checkstyle/checkstyle/assets/23459549/a8e69e68-b494-45aa-9f5a-575523232070)

Reply:
You are right, I looked at 6th example